### ISSUE TITLE
75633070 Church Website link improvement (Eric)

### ISSUE THREAD

**Issue by FreeUKGenIssues** [FreeUKGen/FreeUKRegProductIssues#961]
Issue reported by **REGManager** at 2017-05-25 09:11:10 UTC
Time: 2017-05-25T09:06:11+00:00
Session ID: a9d2f214f654b7a4143559b7442e12c5
Problem Page URL: [/freereg_contents/54082b30eca9eb0248dbf378/show_church](/freereg_contents/54082b30eca9eb0248dbf378/show_church)
Previous Page URL: [https://www.freereg.org.uk/freereg_contents/Marden](https://www.freereg.org.uk/freereg_contents/Marden)
Reported Issue:
Can this link actually work by opening it in a separate Tab?
V1.7?
http://www13.freereg.org.uk/uploads/feedback/screenshot/59269faef493fd763208e740/Capture.JPG    

![Screenshot](http://www.freereg.org.uk/uploads/feedback/screenshot/59269faef493fd763208e740/Capture.JPG)

**Comment by ghost** [comment]
![capture](https://cloud.githubusercontent.com/assets/20557838/26493109/8baaa558-420f-11e7-890a-7c3972c9eb6c.JPG)

**Comment by Captainkirkdawson** [comment]
This goes against the accessibilty guidelines where going to a different tab creates issue?

**Comment by DeniseColbert** [comment]
As I understand the guidelines, links can open in a new tab but we must communicate that to users.

**Comment by Captainkirkdawson** [comment]
How do we do that?

**Comment by DeniseColbert** [comment]
Some ideas here:
https://www.w3.org/TR/WCAG20-TECHS/G201.html

With some more info here:
https://www.w3.org/TR/WCAG20-TECHS/G200.html

**Comment by AlOneill** [comment]
_Or_ ... we could leave it as opening in same tab, but provide brief instructions on how to open in a new tab as the user requires: right-click (Win) or ctrl-click (Mac) for context menu containing "Open link in new tab".

**Comment by Sherlock21** [comment]
Seems like these guidelines make life too much trouble to bother with doing anything helpful to the user.

Lets remove them all. This decision would save me an awful lot of work

**Comment by edickens** [comment]
Agreed.  A researcher, if they are computer literate, can just copy and paste the link into another tab.  ED

**Comment by Captainkirkdawson** [comment]
I believe that EricB was saying remove the link totally if  read his note correctly. But EricD seems to be saying something very different.

PS the link does open in a new tab but with no notification

**Comment by Sherlock21** [comment]
To expand on what I was meaning to say:-

To just show within FR, the url that a user needs to use, to the location map & churches in Genuki, is only feasible if the user is competent to actually do that. Eric D says similar - 
He says: "if the user is computer literate".

My point is that many of out researchers are not so literate as to be able to do that. So providing something that not all our users can use, seems to me to be contrary to your objectives of having a User friendly site.

SO I am saying: make the site useably to the same degree, by all grades of user. OR removed this Location navigation feature for all if you can't put a simple note on the screen: this link opens in the Genuki site, in a new window.. Then that stops all the stupid work rounds.

Its worked file for all users for the past 2 or 3 years or more,  and is the basis of our renowned Nearby tool - not being copied by others sites.

**Comment by Captainkirkdawson** [comment]
Thank you for the clarrification ericb.

The link currently DOES open in a new window. Adding notification can be done.

I like the link and believe it serves a useful capability for our users.

**Comment by Sherlock21** [comment]
Just to be doubly clear:
the url link in the attachment in BOX 2 at the start of this sage is NOT the Navigation url to LOCATION that I am referring to throughout my comments. - the green box on the Placename  details page.

The url quoted in the attachment in box 2 above here, is actually just a typed out  name string - its not a navigable url link. So to use it, the user would have to get up a new browser page to do so.

**Comment by Captainkirkdawson** [comment]
Sorry there is no attachment so do not understand

**Comment by SteveBiggs** [comment]
The "Location" green box is a clickable link that opens in a new tab (at least it does in Chrome), but the church website URL is just plain text and I believe EricB is asking that the URL also be made into a clickable link that opens in a new tab.

**Comment by Captainkirkdawson** [comment]
duh

TY

**Comment by Sherlock21** [comment]
Actually, Steve, I was not asking that which your refer to.

I was just trying to clarify which url embedded in FR was the one to which I understood the original complaint had referred, based on the adverse comments made but others before me, and upon which Pat had made her comment: remove all NI data. - but I now see her comments and others have actually been removed.

And for the record I see the Navigate to LOCATION button as being useful to many, whereas the url of the church site is of interest to some and does not need to be made into a navigable link
